client/app/components/dialog-editor-modal-field-template/dynamic.html

```diff
@@ -0,0 +1,35 @@
+<!-- any dynamic element -->
+<div pf-form-group pf-label="{{'Entry Point'|translate}}">
+  <!-- TODO: until treepicker and API for it are in the SSUI -->
```

Comment:
@jntullo a dynamic button needs to know the entry point. In ui-classic, we show `TreeBuilderAeClass` ([link](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/presenters/tree_builder_ae_class.rb) - should help you see what is actually being accesssed) (`:automate_tree` or `:ae_tree` - depending on whether methods should be displayed as well as instances) to let the user pick that entry point.

One option would be to make the whole tree accessible via the API, which is something we will need to be able to do for all the trees in the future (but not necessarily right now), another option would be to expose all that data via the API, and reimplement the tree on top of that.